BadgeBLE.md

```diff
@@ -2,14 +2,22 @@
 
 ### Connection Information
 
-The pixel data is transferred to the badge via BLE. The badge advertizes the name "LSLED" and uses a vendor specific service/characteristic:
+#### OEM Firmware
 
-- Service-UUID: 0000fee0-0000-1000-8000-00805f9b34fb
-- Characteristic: 0000fee1-0000-1000-8000-00805f9b34fb
+The pixel data is transferred to the badge via BLE. The OEM firmware advertizes the name "LSLED", and a 16-bit vendor specific service/characteristic:
 
-The characteristic accepts 16 byte long data packets.
+- Service-UUID: 0xFEE0 (128-bit equivalent: 0000fee0-0000-1000-8000-00805f9b34fb)
+- Characteristic: 0xFEE1 (128-bit equivalent: 0000fee1-0000-1000-8000-00805f9b34fb)
 
-### Data Format
+The characteristic of the OEM firmware has Read/Write/Notify properties. It accepts 16 byte long data packets on writing. Its Read/Notify behavior currently unknown, to be investigated.
+
+#### Open Firmware
+
+This open firmware advertizes service/characteristic 0xFEE0/0xFEE1 as the same as the OEM firmware, and the name "LED Badge Magic" which can be renamed by the [next-gen profile](#next-gen-profile). The characteristic 0xFEE1, marked as `legacy` profile, has only Write propertie. Read/Notify will be supported after its behavior discovered.
+
+Additional custom profile has been implemented, enabling more remote features and configuration over BLE connection, see [next-gen profile](#next-gen-profile).
```

Comment:
An additional custom profile has been implemented, enabling more remote features and configuration over the BLE connection. See [next-gen profile](#next-gen-profile) for details.